ts/features/itwallet/presentation/components/ItwCredentialTrustmark.tsx

```diff
@@ -51,15 +80,204 @@ const shouldDisplayTrustmark = (credential: StoredCredential) =>
     credential.credentialType as CredentialType
   ) && getCredentialStatus(credential) !== "expired";
 
+/* VISUAL PARAMETERS */
+
+/* Button */
+const TRUSTMARK_HEIGHT = 48;
+const TRUSTMARK_STAMP_SIZE = TRUSTMARK_HEIGHT * 1.75;
+const TRUSTMARK_STAMP_SIZE_RASTER = TRUSTMARK_HEIGHT * 1.5;
+const TRUSTMARK_GRADIENT_HEIGHT = TRUSTMARK_STAMP_SIZE * 2;
+const buttonBorderRadius = 12;
+const buttonInnerBorderColor: ColorValue = "#CCCCCC";
+const buttonBackgroundGradient = {
+  colors: ["#CCCCCC", "#F2F2F2", "#E9E9E9", "#E0E0E0"],
+  locations: [0, 0.35, 0.7, 1],
+  center: { x: 0.5, y: 0.7 }
+};
+
+/* Light */
+const lightScaleMultiplier = 1;
+const lightSkiaOpacity = 0.7;
+const lightSize: LayoutRectangle["width"] = 250;
+const visibleLightPercentage = 0.25; // Visible light when it's near box boundaries
+
 export const ItwCredentialTrustmark = ({
   testID,
   credential
 }: ItwCredentialTrustmarkProps) => {
+  /* Bottom sheet for the QR code */
   const trustmarkBottomSheet = useIOBottomSheetAutoresizableModal({
     title: I18n.t("features.itWallet.presentation.trustmark.title"),
     component: <QrCodeBottomSheetContent credential={credential} />
   });
 
+  /* Enable the effect only when the experimental DS is enabled */
+  const { isExperimental: enableIridescence } = useIOExperimentalDesign();
+
+  const rotationSensor = useAnimatedSensor(SensorType.ROTATION);
+  const initialQx = useSharedValue(0);
+  const { qx } = rotationSensor.sensor.value;
```

Comment:
Why was `qx` chosen instead of `qy`? I believe the effect would feel more intuitive if it followed the device's roll rather than pitch. It may provide a more "pleasant" effect.